src/scss/constants.scss

```diff
@@ -203,6 +209,8 @@ $z-layers: (
   account-profile-picture-preview: 1,
   duststorm-container: 100,
   duststorm-modal-content: 220,
+  // Announcement banner
+  announcement-banner: $z-layer-announcement-banner,
   // Unsorted
   announcement: $z-layer-sidebar,
```

Comment:
if we add `announcement-banner`, should we remove `announcement`?

Reply:
done ed799a1